.bazelversion

```diff
@@ -1 +1 @@
-4.2.2
+5.0.0
```

Comment:
[Ladies and gentlemen, this is Mambo Number Five](https://www.youtube.com/watch?v=EK_LN3XEcnw)